src/firmware_new/boards/frankie_v1/stm32h7xx_hal_msp.c

```diff
@@ -152,16 +152,29 @@ void HAL_TIM_MspPostInit(TIM_HandleTypeDef* htim)
 
         /* USER CODE END TIM4_MspPostInit 0 */
 
+        __HAL_RCC_GPIOD_CLK_ENABLE();
         __HAL_RCC_GPIOB_CLK_ENABLE();
         /**TIM4 GPIO Configuration
+        PD13     ------> TIM4_CH2
+        PD14     ------> TIM4_CH3
+        PD15     ------> TIM4_CH4
         PB6     ------> TIM4_CH1
         */
-        GPIO_InitStruct.Pin       = motor_0_pwm_Pin;
+        GPIO_InitStruct.Pin = wheel_motor_back_left_pwm_Pin |
+                              wheel_motor_back_right_pwm_Pin |
+                              wheel_motor_front_right_pwm_Pin;
```

Comment:
Just curious why `wheel_motor_front_left_pwm_Pin` didn't get generated here? Its got a seperate section below, whats different about that motor?

Reply:
Great question! There's nothing particuarly special about that motor, but it's on pin `PB6`, where as all the other ones are on `PD*` pins. I'm not entirely certain of the lower level details, but these are basically different "groups" of GPIO pins.